scripts/configure-dh.sh

```diff
@@ -131,6 +131,30 @@ fi
 
 echo "OK"
 
+# Reads Model URL for lightspeed plugin
+# Optional: If no URL is entered, lightspeed plugin will not be configured
+if [ -z "${LIGHTSPEED_MODEL_URL}" ]; then
+    read -p "Enter your model URL for lightspeed (Optional): " LIGHTSPEED_MODEL_URL
+
+    # Reads API token for lightspeed plugin
+    # Optional: If no token is entered, lightspeed plugin will not use authenticated communication
+    if [ -z "${LIGHTSPEED_API_TOKEN}" ]; then
+        read -p "Enter API token for lightspeed (Optional): " LIGHTSPEED_API_TOKEN
```

Comment:
Nit: Would be good to add `LIGHTSPEED_MODEL_URL` and `LIGHTSPEED_API_TOKEN` to the secret variable definitions above.